### PR TITLE
4docs: bring styles in line with Zanata app styles

### PR DESCRIPTION
**Pull request by carlosmunoz**
This is a documentation-only change.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -2,7 +2,11 @@ site_name: Zanata Server
 docs_dir: docs
 repo_name: GitHub
 repo_url: http://www.github.com/zanata/zanata-server
+site_favicon: images/favicon.ico
```
Comment (aeng) [inline mkdocs.yml]: Should we use the one from assets? http://assets-zanata.rhcloud.com/master/favicon.ico

File: mkdocs.yml [hunk 2]
```diff
@@ -2,7 +2,11 @@ site_name: Zanata Server
...
 theme: readthedocs
+extra_css:
+  - css/extra.css
+  - 'http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic'
```
Comment (aeng) [inline mkdocs.yml]: Have you try to bring in http://assets-zanata.rhcloud.com/master/assets/css/style.css and see what the page looks like? trying not to introduce another css.

File: mkdocs.yml (hunk 1 above)
Comment (carlosmunoz) [inline mkdocs.yml]: It actually is (a copy of) that one. I'm not sure mkdocs can take a url for the favicon, but let me check. If not, what's your take on either 1. having a copy of the file here, or 2. Not using a Zanata favicon.

File: mkdocs.yml (hunk 2 above)
Comment (carlosmunoz) [inline mkdocs.yml]: Yeah, that's a good idea... as long as that one has the font styles in it. I'll give it a try.

**Comment by carlosmunoz** [comment]
@aeng So it turns out the favicon **can** be used from a url, however, using the css from assets did not change the font. Still, using a css from a common external site is not a big problem.

**Comment by aeng** [comment]
Think to use the font in assets css, it will need to be 

`extra_css:
  -  http://assets-zanata.rhcloud.com/master/assets/css/style.css
  - 'http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic'`

Also, need to include class `new-zanata` in the <html> tag.

**Comment by aeng** [comment]
:+1: if its too problematic to use external css and apply the `new-zanata` class.

**Comment by definite** [comment]
@zanata-jenkins retest this please

**Comment by anipeter** [comment]
@zanata-jenkins retest this please

**Comment by anipeter** [comment]
Looks like something is broken, possible need to rebase

**Comment by seanf** [comment]
I believe that problem (jboss-jaxrs-api_2.0_spec) has been fixed now.

@zanata-jenkins retest this please

**Comment by carlosmunoz** [comment]
> :+1: if its too problematic to use external css and apply the new-zanata class.

adding external css is possible, but adding the `new-zanata` class is not for markdown.

**Comment by carlosmunoz** [comment]
@zanata-jenkins retest this please

**Comment by definite** [comment]
@zanata-jenkins retest this please

**Comment by carlosmunoz** [comment]
@anipeter auto-tests passed. Is this ok to merge?

**Comment by anipeter** [comment]
@carlosmunoz looks good. 
Acknowledging change in font : colour and size; zanata icon in the window on home page. 
There is a problem with the order of docs while using the navigation buttons. Filing separate bug for that issue

**Pull request merged**